### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by TaserTheFox** [TaserTheFox/EsoFur-Interpreter#16] [Announcements]
Welcome to the discussions section of this repo that no one will likely ever see, I have no idea why I'm making this but who cares?!

I'll probably occasionally update this with new features... if any...

**Comment by madeforlosers** [comment]
this is the best programming language ever

**Reply by TaserTheFox** [reply]
thank you! i’m glad you enjoyed it, or at least had a giggle at it

where did you find this if i can ask by the way?

**Reply by madeforlosers** [reply]
i found it on esolangs going through random pages lol. i made a collatz conjecture program in it

**Reply by madeforlosers** [reply]
its actually a cool esoteric language (at least i find it cool) and works pretty well (for the stuff that is supposed to work)

it'd be funny if someone made a game with it x)
